components/user/ChangeEmail.vue

```diff
@@ -7,21 +7,28 @@
         class="flex flex-col p-6 pt-0 space-y-2"
         @submit.prevent="addNewEmail"
       >
-        Enter the new email you wish to switch to, and your password.
         <text-input
           v-model="email"
           name="New Email"
           type="email"
+          placeholder="New Email"
           :errors="errors"
           required
         />
         <text-input
           v-model="password"
           name="Password"
           type="password"
+          placeholder="Password"
           :errors="errors"
           required
         />
+        <div
+          v-if="errors && errors.has('email')"
+          class="text-sta-rouge text-sm font-semibold"
+        >
+          <p>Error: {{ errors.first('email').message }}</p>
+        </div>
```

Comment:
This should be automatically handled by the `text-input` component - check out https://github.com/BristolSTA/uobtheatre-web/blob/develop/components/ui/Input/UiInputText.vue. I think you just need to specify `error-key` attribute on each of them